src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/TableSaw.java

```diff
@@ -46,6 +51,11 @@ public TableSaw(Category category, SlimefunItemStack item) {
                 displayedRecipes.add(new ItemStack(planks.get(), 8));
             }
         }
+
+        CustomItem ci = new CustomItem(Material.OAK_PLANKS, "Any Wooden Plank");
+
+        displayedRecipes.add(new ItemStack(ci));
+        displayedRecipes.add(new ItemStack(Material.STICK, 4));
```

Comment:
Why aren't you adding all planks directly instead of this weird placeholder?

Reply:
I dont know who it was but someone told me that this was better, I had it for each item at the start and changed it to this

Reply:
undo or keep?

Reply:
I undid cuz I also preferred the older way of doing it